config/protocol.unit_testnet.yml

```diff
@@ -18,7 +18,7 @@ ProtocolConfiguration:
   VerifyTransactions: true
   P2PSigExtensions: true
   Hardforks:
-    Aspidochelone: 25
+#    Aspidochelone: 25
```

Comment:
We can't do it. We have to keep hardforks scheduler because otherwise all hardforks will be enabled by default. It leads to the fact that every time we introduce new hardfork, unit-tests may break somewhere.

You may enable all of them, but use strictly-defined height for every fork.